frontend/src/pages/Alerts/ErrorAlert/ErrorAlertPage.tsx

```diff
@@ -82,7 +82,7 @@ export const ErrorAlertPage = () => {
 		? (findAlert(alert_id, alertsPayload) as any)
 		: undefined
 
-	const form = useFormState<ErrorAlertFormItem>({
+	const formStore = useFormStore<ErrorAlertFormItem>({
```

Comment:
the alerts input box seems to have a blue focus box for me
![image](https://github.com/highlight/highlight/assets/1351531/a9aba90a-b16b-470b-8b73-ce09d23dcd6c)

Reply:
I'm only seeing this in the Reflame preview for some reason 🤔 - not sure why this is happening, but don't think it's something that will make it to prod.

Reply:
I think this has something to do with Tailwind styles being applied that aren't applied in our app.

<img width="1412" alt="Screenshot 2023-09-07 at 7 47 36 AM" src="https://github.com/highlight/highlight/assets/308182/face86e5-c771-4dde-8a8e-11378f16d059">

Reply:
yeah, sorry for the false alarm - not an issue with the render.com preview. i think it's the same reflame issue that shows tge old gradient styling for the harold box (only in reflame). maybe some CSS is outdated / cached? cc @lewisl9029

Reply:
Ah yep, this definitely looks like a bug with Reflame's Tailwind compat. 

We're actually using UnoCSS under the hood since Tailwind itself is way too slow to fit in our performance budget (their new rust based tooling looks promising but isn't ready for prime time yet).

Will look into it and try to make the output closer to real Tailwind here!